src/components/rangeslider/defaults.js

```diff
@@ -48,4 +52,7 @@ module.exports = function handleDefaults(layoutIn, layoutOut, axName, counterAxe
             layoutOut[ax] = opposing;
         });
     }
+
+    // to map back range slider (auto) range
+    containerOut._input = containerIn;
```

Comment:
`_input` is always linked to the `layout` as passed to `Plotly.plot(gd, data, layout)` - which doesn't know anything about `containerOut`.